recipes/ncbi-cxx-toolkit-public/26.0/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.7)
+project(PackageTest)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+if(OFF)
+  add_executable(basic_sample basic_sample.cpp)
+  target_link_libraries(basic_sample ${CONAN_LIBS})
+else()
+  NCBI_begin_app(basic_sample)
+    NCBI_sources(basic_sample)
+    NCBI_uses_toolkit_libraries(xncbi)
+  NCBI_end_app()
+endif()
```

Comment:
Maybe it is useful to use both and generate two different `basic_sample` and `basic_sample1` applications, wdyt?

Reply:
I have now updated test_package files.
The only problem is that variant with "conan_basic_setup(TARGETS)" is not working. In fact, NCBI toolkit calls this function itself. And calling it two times is not allowed.